packer/gpu/scripts/install-nvidia-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# nvidia driver
+sudo yum install -y kernel-devel-$(uname -r) kernel-headers-$(uname -r) dkms
```

Comment:
could you document what user does this script run as?

Reply:
I meant liunx user name / id, like if it is root, if it is not, what user it is?

my guess would be that it is amazon linux's default admin non-root user, maybe `ec2-user` 

it is for justify the use of sudo, and make it clear on the ownership of the dowloaded files if any.